### PR TITLE
Add QuoteOfTheDay examples for experimentation with feature managment

### PR DESCRIPTION
**Pull request by amerjusupovic**

### CODE REVIEW

File: examples/Experimentation/Dotnet/QuoteOfTheDayQuickStart/README.md
Comment (zhenlan) [inline examples/Experimentation/Dotnet/QuoteOfTheDayQuickStart/README.md:1]: Can we add a brief introduction of the app? Something like this?

The _Quote of the Day_ app displays a quote upon starting. Users can hit the heart button to like the quote. The app displays a standard title for some users, while other users will receive a greeting message based on the targeting rules defined in a feature flag. The feature experimentation (aka. A/B testing) is to measure whether a personalized greeting message can promote more user engagement by liking the quote.

[the remark this answers is not in the document]
Comment (amerjusupovic) [reply examples/Experimentation/Dotnet/QuoteOfTheDayQuickStart/README.md:1]: do we still need an example image? I didn't add one yet

[the remark this answers is not in the document]
Comment (zhenlan) [reply examples/Experimentation/Dotnet/QuoteOfTheDayQuickStart/README.md:1]: Not required, but I think it will be nice to have one.

**Comment by zhenlan** [comment]
The example is published at https://github.com/Azure-Samples/quote-of-the-day-dotnet. Close the PR.

**Pull request closed without merging**